src/distros/arch/installer.py

```diff
@@ -16,6 +16,22 @@ def initram_update_luks():
                         -e 's|^FILES=(|FILES=(/etc/crypto_keyfile.bin|' /mnt/etc/mkinitcpio.conf")
         os.system(f"sudo chroot /mnt sudo mkinitcpio -p linux{KERNEL}")
 
+def pacstrap(pkg):
+    while True:
+        excode = int(os.system(f"sudo pacstrap /mnt --needed {pkg}"))
```

Comment:
Can you make this a generic distro-agnostic function? :
- Given only line 21 distro-specifc, put this in installer_core.py with a call for that line in installer.py
- Change function names to `download` and `download_helper` (like examples in ashpk.py) - something less Archy
- Use try except else instead